.circleci/config.yml

```diff
@@ -1,24 +1,114 @@
 version: 2.1
 orbs:
-  ruby: circleci/ruby@0.1.2 
+  ruby: circleci/ruby@0.1.2
+
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - test
+      - build
+
+defaults:
+  ruby_image: &ruby_image
+    circleci/ruby:2.5.5-stretch-node
+
+  executor: &executor
+    executor: ruby/default
+
+  restore_cache: &restore_cache
+    keys:
+      - gem-cache-v1-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+      - gem-cache-v1-{{ arch }}-{{ .Branch }}
+      - gem-cache-v1
+
+  install_cmake: &install_cmake
+    name: Install cmake
+    command: |
+      sudo apt-get update
+      sudo apt-get install cmake
+
+  install_dependencies: &install_dependencies
+    name: Install dependencies
+    command: |
+      gem update --system
+      gem i bundler -v $(tail -1 Gemfile.lock | tr -d ' ')
+      bundle install --path vendor/bundle --jobs 4 --retry 3
+
+  save_cache: &save_cache
+    key: gem-cache-v1-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+    paths:
+      - vendor/bundle
 
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.5.5-stretch-node
-    executor: ruby/default
+      - image: *ruby_image
+    <<: *executor
     steps:
       - checkout
-      - run:
-          name: Install dependencies
-          command: |
-            gem update --system
-            gem install bundler
-            bundle install --path vendor/bundle
+      - restore_cache:
+          <<: *restore_cache
+      - run: 
+          <<: *install_cmake
+      - run: 
+          <<: *install_dependencies
+      - save_cache:
+          <<: *save_cache
       - run:
           name: Running checks
           command: |
             git config --global user.email "circle@example.com"
```

Comment:
Needless. You can use `SKIP=AuthorEmail,AuthorName` instead.

Reply:
I've checked it and it isn't worked

Reply:
Could you try with this options?

.overcommit.yml:
```
PreCommit:
  AuthorEmail:
    enabled: true
    required: false

  AuthorName:
    enabled: false
```

config.yml:
```
- run:
       name: Running overcommit
       command: |
            bundle exec overcommit -s
            SKIP=AuthorEmail,AuthorName bundle exec overcommit -r
```